### PR TITLE
Fix default values in getPublicFields()

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

**Comment by amitaibu** [comment]
@RoySegall please take this from here.

**Comment by RoySegall** [comment]
@amitaibu Any thoughts on this one? This one look fine to me.

File: includes/EntityValidateBase.php [hunk 1]
```diff
@@ -278,6 +280,11 @@ protected function isValidValue($field_name, $value, EntityMetadataWrapper $wrap
       return;
     }
 
+    if (is_array($value)) {
```
Comment (amitaibu) [inline includes/EntityValidateBase.php]: instead move the `$this->setError()` above.

**Comment by RoySegall** [comment]
For some reason the multiple text field is detect in `entity_property_verify_data_type` once as an array and everything OK and in another time it's detect as a string and there is a problem. We need to see why.

**Comment by RoySegall** [comment]
Merged.

**Pull request closed without merging**